widget/list.go

```diff
@@ -290,7 +291,7 @@ func (l *List) handleInput() {
 		}
 
 		l.buttons[l.focusIndex].focused = true
-	} else {
+	} else if len(l.buttons) > 0 {
```

Comment:
If we're trying to avoid null pointers here, and above, perhaps instead of checking vs 0 we should check vs the focusIndex.  So `if l.focusIndex <= len(l.buttons) {`